docs/dev_guide/diagnostic_ref/dpct1017.rst

```diff
@@ -25,4 +25,37 @@ code precision and/or performance. Verify the code correctness/performance.
 Suggestions to Fix
 ------------------
 
-Verify the code correctness/performance.
\ No newline at end of file
+Verify the code correctness/performance.
+
+For example, this original CUDA\* code:
+
+.. code-block:: cpp
+   :linenos:
+
+   __global__ void k(float *out, float *in) {
+     *out = normf(3, in);
+   }
+
+results in the following migrated SYCL code:
+
+.. code-block:: cpp
+   :linenos:
+
+   void k(float *out, float *in) {
+     /*
+     DPCT1017:0: The dpct::length call is used instead of the normf call. These two
+     calls do not provide exactly the same functionality. Check the potential
+     precision and/or performance issues for the generated code.
+     */
+     *out = dpct::length(in, 3);
+   }
+
+which is rewritten to:
+
+.. code-block:: cpp
+   :linenos:
+
+   #include <sycl/ext/intel/math.hpp>
+   void k(float *out, float *in) {
+     *out = sycl::ext::intel::math::norm(3, in);

```

Comment:
@zhiweij1 , please add comment to explain why 60 works ( why not use migrate to line60 directly).

Reply:
Because it is an extension. If user did not specify the dpct option, Line 50 will be the result.
If users make sure the entension is available, they can change it to extension version manually.

Reply:
Added.